### ISSUE TITLE
Can't open task in JOSM (too much data?)

### ISSUE THREAD

**Issue by aawiseman** [hotosm/osm-tasking-manager2#782]
This task http://tasks.hotosm.org/project/1755#task/245 won't open in JOSM, but others nearby will -- maybe there's too much data? I think a solution would be letting you split some areas farther if they're in urban areas -- or just letting you split squares above a certain size.

**Comment by ethan-nelson** [comment]
Yes it looks like there is way too much data in this square. I think this unfortunately falls back on the project manager to ensure tiles are a manageable size (though you can use the gpx file I believe and download in JOSM using that--or so I hear). I was looking to add the capability to prevent any splitting based on another filed issue, so perhaps this can be a sliding scale in the project management screen:

`Set how many times tiles can be split: 0 1 2 3 4 5 6 7`

or something like that.

**Comment by harry-wood** [comment]
Yes a mapper could use the GPX file and use that as a guide while they download the square bit by bit. That works, but it's a clumsy process.

I agree project managers should really avoid this problem by ensuring task squares are a manageable size. Mind you, this square probably _become_ less manageable as the mapping progressed, so now validators can't open the square easily.

Those with permission to do so, can solve this by splitting the square again at this stage (except then we'd lose the history. Bing on this PR https://github.com/hotosm/osm-tasking-manager2/pull/781 )
